### PR TITLE
Main

### PR DESCRIPTION
**Pull request by tcannonfodder**

### CODE REVIEW

File: xbox360controller/controller.py [hunk 1]
```diff
@@ -298,8 +298,33 @@ def get_event(self):
                     time=time_, type=type_, number=number, value=value, is_init=is_init
                 )
 
+    def axis_callback(self, axis, val):
+        if (
+                axis.when_moved is not None
+                and abs(val) > self.axis_threshold
+                and callable(axis.when_moved)
+            ):
+                axis.when_moved(axis)
```
Comment (linusg) [inline xbox360controller/controller.py:307]: This isn't indented correctly, please use black to format the code.

**Review by linusg — CHANGES_REQUESTED** [review]
I appreciate the attempted contribution, but dumping some code without a PR message ("Main"?!) or useful commit messages is really not helpful. Please explain what problem this is trying to solve.

**Comment by tcannonfodder** [comment]
Hey Linus!

Really sorry about that! I guess I opened the PR accidentally while trying to hack together a prototype last night. I’ll close this PR and open another one with a clean commit history :)

**Comment by tcannonfodder** [comment]
Closing because it was accidentally opened, will re-submit with a clean history

**Pull request closed without merging**